src/main/java/draylar/goml/compat/DynmapCompat.java

```diff
@@ -0,0 +1,189 @@
+package draylar.goml.compat;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import draylar.goml.api.Claim;
+import draylar.goml.api.ClaimUtils;
+import draylar.goml.api.event.ClaimEvents;
+import eu.pb4.polymer.core.api.block.PolymerHeadBlock;
+import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.text.Text;
+import net.minecraft.util.math.Box;
+import org.dynmap.DynmapCommonAPI;
+import org.dynmap.DynmapCommonAPIListener;
+import org.dynmap.markers.AreaMarker;
+import org.dynmap.markers.MarkerAPI;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import static draylar.goml.GetOffMyLawn.CLAIM;
+
+public class DynmapCompat {
+    private static final String gomlMarkerSetId = "gomlMarkerSet";
+
+    // From https://lospec.com/palette-list/minecraft-concrete (matches block order so matches goggles).
+    private static final int[] COLORS = new int[]{0xcfd5d6, 0xe06101, 0xa9309f, 0x2489c7, 0xf1af15, 0x5ea918, 0xd5658f, 0x373a3e, 0x7d7d73, 0x157788, 0x64209c, 0x2d2f8f, 0x603c20, 0x495b24, 0x8e2121, 0x080a0f};
+
+    /**
+     * Store all players uuids and names to update existing claims when a player changes its name.
+     * This value gets reset each time the server stops, but it really doesn't matter.
+     */
+    private static final HashMap<UUID, String> playerNames = new HashMap<>();
```

Comment:
So you're saying it would be best to get rid of this as well?

Reply:
Yep, since the server already keeps that data, it would just a useless duplicate

Reply:
I will fix it in a few hours and continue my testing with this change 👍🏼

Reply:
Any idea on how to replace the `!playerNames.containsKey(uuid) || !playerNames.get(uuid).equals(name)` check?  
I would like to keep it for performance reasons, to avoid refreshing all of the claims related to a user every time they join.  
At the same time, I'm not certain whether I can use the server userCache in place of the playerNames map because I don't know whether that cache is update before the `ServerPlayConnectionEvents.JOIN` event.

Reply:
Usercache is generally loaded/kept since player joined the server for the first time (or server requested uuid-name of the player), so it should be safe to use (and it can do mojang api lookups, making it usable for players not in cache)

Reply:
Yeah, but if the cache is update before that event, then I don't know how to check whether user related claims need an update or not.

Reply:
Vanilla already has special handling for name changes in `PlayerManager#onPlayerConnect`, so you could just hook up there